Package.swift

```diff
@@ -14,7 +14,7 @@ let package = Package(
         .package(url: "https://github.com/vapor/fluent.git", from: "3.0.0"),
 
         // 🐬 Pure Swift MySQL client built on non-blocking, event-driven sockets.
-        .package(url: "https://github.com/vapor/mysql.git", from: "3.0.0"),
+        .package(url: "https://github.com/vapor/mysql.git", .revision("8cf855487d6dc011449cf9c565c295ed24315a75")),
```

Comment:
This will need reverting

Reply:
@0xTim yes... this is just to make the tests pass, because this need the changes of mysql package... The question is, reverting now or when a new mysql be tagged?